integrationtests/server-integration/third-party-server/pom.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-server-integration</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>infinispan-server-integrationtests-third-party</artifactId>
+
+    <properties>
+        <version.tomcat>9.0.37</version.tomcat>
+        <version.wildfly>20.0.1.Final</version.wildfly>
+        <version.arquillian-tomcat-embedded>1.1.0.Final</version.arquillian-tomcat-embedded>
```

Comment:
Here looks like we will use tomcat-embedded but we are using tomcat-managed, IMO `version.arquillian-tomcat-managed` sounds better